releasenotes/notes/backend-run-shots-4b73fb06470688e9.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    Floats can now be use when setting the number of ``shots`` 
+    in :meth:`qiskit_ibm.IBMBackend.run`.
```

Comment:
```suggestion
    Floats can now be used when setting the number of ``shots`` 
    in :meth:`qiskit_ibm.IBMBackend.run`. Now instead of having to type
    larger values like `100000` you can just do `1e5`.
```